base/strings/sys_string_conversions_posix.cc

```diff
@@ -27,7 +27,7 @@ std::wstring SysUTF8ToWide(StringPiece utf8) {
   return out;
 }
 
-#if defined(SYSTEM_NATIVE_UTF8) || BUILDFLAG(IS_ANDROID)
+#if defined(SYSTEM_NATIVE_UTF8) || BUILDFLAG(IS_ANDROID) || defined(STARBOARD)
```

Comment:
Should we be defining `SYSTEM_NATIVE_UTF8`?

Reply:
I agree, that's a great suggestion. I've reviewed the relevant call sites and found that if SYSTEM_NATIVE_UTF8 isn't defined, we're ultimately performing additional type conversions in this section of code. We should just define this.